packages/store-sync/src/recs/syncToRecs.ts

```diff
@@ -59,9 +68,61 @@ export async function syncToRecs<TConfig extends StoreConfig = StoreConfig>({
 
   world.registerDisposer(stopSync);
 
+  const functionSelectorToResourceSelector = new Map<Hex, Hex>();
+
+  const getResourceSelector = async (functionName: string): Promise<Hex> => {
+    const functionSignature = getAbiItem({
+      abi,
+      name: functionName,
+    }) as AbiFunction;
```

Comment:
instead of passing the ABI around, I wonder if `getResourceSelector` should take in `functionSignature` or `functionSelector` as an argument (rather than function name) and we do the `functionSignature`/`functionSelector` calculation inside `createContract` (where we already have the ABI)

Reply:
Agree - but abi is also needed to make the getRecord rpc call. I can move the abi to be a param of getResourceSelector instead of syncToRecs

Reply:
since we're doing a generic Store call (`getRecord` comes from `Store`), we could use the store ABI for that (available as an import)